sk0-front/src/app/auth/auth.service.ts

```diff
@@ -1,22 +1,23 @@
 import { Injectable } from '@angular/core';
 import { AlertController } from '@ionic/angular';
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { environment } from 'src/environments/environment';
+import { BehaviorSubject } from 'rxjs';
+import { NavController } from '@ionic/angular';
 
-type User = {
-  name: string;
-  email: string;
-  password: string;
-};
+interface LoginResponse {
+    name: string;
+    email: string;
+    id: number;
+}
```

Comment:
What is the reason you used interface instead of type here?

And 4 spaces? -> we should add a linter to avoid opinionated coding styles.

Reply:
I simply prefer the use of the interface in the OOP field. Adding a linter would be a useful thing I think